pkgs/development/python-modules/array-api-strict/default.nix

```diff
@@ -37,13 +36,11 @@ buildPythonPackage rec {
     "test_environment_variables"
   ];
 
-  passthru.updateScript = nix-update-script { };
```

Comment:
Why?

Reply:
The review in https://github.com/NixOS/nixpkgs/pull/334997#discussion_r1726738619 said that it isn't necessary.

Reply:
Yep, that's correct. There is already a default update script.